docs/pages/database-access/guides/azure-sql-server-ad.mdx

```diff
@@ -0,0 +1,183 @@
+3. Select **Set Admin**, and choose an account that will be added as an admin
+   login to SQL Server.
+
+![Azure SQL Server Active Directory admin page](../../../img/database-access/guides/sqlserver/azure-set-ad-admin.png)
+
+## Step 4/7. Configure Azure Managed Identities
+
+Teleport Database Service needs access tokens from Azure AD to authenticate with
+SQL Server databases.
+
+It uses the managed identities attached to its Virtual Machine to fetch the
+authentication token.
+
+To create a new user-assigned managed identity, go to the **Managed Identities**
+page in your [Azure Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.ManagedIdentity%2FuserAssignedIdentities)
+and click on *Create*. Choose a name and resource group for it and create:
+
+![Azure Create user managed identity page](../../../img/database-access/guides/sqlserver/azure-user-managed-identity.png)
+
+Next, go to the **Teleport Database Service virtual machine instance**,
+**Identity** section, select **User assigned**, and add the identity we just
+created:
+
+![Azure Virtual machine user managed identities page](../../../img/database-access/guides/sqlserver/azure-attach-managed-identity-vm.png)
+
+## Step 5/7. Enable managed identities login on SQL Server
+
+Connect to your SQL Server using a pre-existent authentication method or the
+Activity Directory Admin set on
+[Step 3](#step-37-enable-sql-server-active-directory-integration).
+
+Create a [contained user](https://learn.microsoft.com/en-us/sql/relational-databases/security/contained-database-users-making-your-database-portable?view=sql-server-ver16)
+for your identities:
+```sql
+CREATE USER [sqlserver-identity] FROM EXTERNAL PROVIDER;
+```
+
+The newly created login will be attached to the public role, which might not
+have enough permissions to perform queries. Consider granting individual
+permissions or adding it to an existing role. For example, adding to
+`db_datareader` role:
+```sql
+ALTER ROLE db_datareader ADD MEMBER [sqlserver-identity];
+```
+
+<Admonition type="note">
+Since we’re creating contained users, this step needs to be done on each
+database we want to be able to log in.
+</Admonition>
+
+## Step 6/7. Start Teleport Database Service
+
+Generate a configuration file at `/etc/teleport.yaml` for the Database Service:
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=teleport.example.com:3080 \
+   --name=sqlserver \
+   --protocol=sqlserver \
+   --uri=my-server.database.windows.net:1433 \
+   --labels=env=dev
+```
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  A single Teleport process can run multiple different services, for example
+  multiple Database Service agents as well as the SSH Service or Application
+  Service.
+  
+</Admonition>
+
+Start the database service:
+
+```code
+$ teleport start --config=/path/to/teleport-db.yaml --token=/tmp/token
```

Comment:
```suggestion
$ teleport start --config=/etc/teleport.yaml
```
nit: I don't think they need to specify the token again since it should be in the config they generated, and if they used `-o file` then the config will be at /etc/teleport.yaml